keyboards/specskeys/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+F_CPU = 8000000
+
+
+# Bootloader selection
+BOOTLOADER = lufa-dfu
```

Comment:
Is this correct?

Reply:
I updated this. I did not use this feature, I flashed the MCU with avrdude.